test/tests/positions/reporter_positions.i

```diff
@@ -0,0 +1,42 @@
+[Mesh]
+  [cmg]
+    type = CartesianMeshGenerator
+    dx = 1
+    dim = 1
+  []
+[]
+
+[Positions]
+  [input]
+    type = InputPositions
+    positions = '0 0 1
+                 1 0 2'
+    outputs = none
+  []
+  [input_2]
+    type = InputPositions
+    positions = '0.1 0.1 1
+                 1.2 0 2.2'
+    outputs = none
+  []
+  [reporter]
+    type = ReporterPositions
+    reporters = 'input/positions_1d input_2/positions_1d'
+  []
```

Comment:
Dumb question. I thought there was only ever one `Positions` object based on the assertion in `FEProblemBase`?

Reply:
Oh never mind. It was just one per name